pkg/options/start.go

```diff
@@ -669,3 +680,30 @@ func askParam(ques string, askOpts survey.AskOpt, def ...string) (string, error)
 
 	return ans, nil
 }
+
+func getTaskRunV1beta1(gr schema.GroupVersionResource, c *cli.Clients, trName, ns string) (*v1beta1.TaskRun, error) {
```

Comment:
can you please add a comment for why are we converting the v1.TaskRun to v1beta1.TaskRun?